docs/digital-object-full-view.md

```diff
@@ -0,0 +1,32 @@
+# View Modes: Digital Object Full View
+
+`View modes` tell Archipelago Digital Objects (ADOs) how to be displayed to the world. This documentation will give a brief overview of the View mode `Digital Object Full View` and how it can be used when creating or editing ADOs.
+
+## Digital Object Full View
+The `Digital Object Full View` is the default `View mode` used for the Media type `Photograph` and any other Media type that is not already configured to a View mode*. `Digital Object Full View` has the ability to zoom in, zoom out, full screen, rotate and display multiple images associated with an ADO.
```

Comment:
mapped instead of configured. Here is where the mapping piece needs to be clear given the fact that it only kicks in, if the user does not set one manually during ingest (in other words if the 'default' one is used)